packages/core-ethereum/src/indexer/index.ts

```diff
@@ -76,7 +76,7 @@ class Indexer implements IIndexer {
     try {
       return u8aToNumber(
         (await this.connector.db.get(Buffer.from(this.connector.dbKeys.ConfirmedBlockNumber()))) as Uint8Array
-      )
+      ) as number
```

Comment:
why isn't the return type of u8aToNumber a number?

Reply:
If you do call with an array whose number is too large for Number, it will be a BigInt